azure-pipelines.yml

```diff
@@ -1,26 +1,11 @@
-parameters:
-  - name: testProjects
-    type: object
-    default:
-      - OmniSharp.DotNetTest.Tests
-      - OmniSharp.MSBuild.Tests,OmniSharp.Roslyn.CSharp.Tests,OmniSharp.Cake.Tests,OmniSharp.Script.Tests,OmniSharp.Stdio.Tests,OmniSharp.Http.Tests,OmniSharp.Tests,OmniSharp.Lsp.Tests
-
 trigger:
   batch: "true"
   branches:
     include:
       - master
       - refs/tags/*
 
-pr:
-  autoCancel: "true"
-  branches:
-    include:
-      - master
-      - feature/*
-  paths:
-    exclude:
-      - 'CHANGELOG.md'
+pr: none
```

Comment:
We already have a "build" pipeline in GH actions which performs the same steps that this pipeline did for PRs.

Reply:
Branch protections rules have been updated accordingly.